docs/dev_guides/changing_splink/testing.md

```diff
@@ -101,11 +103,14 @@ pytest -W ignore -q -x -m duckdb tests/test_estimate_prob_two_rr_match.py
 
 ??? tip "Running tests with docker 🐳"
 
-    If you want to test Splink against a specific version of python, the easiest method is to utilise docker 🐳.
+    Testing Splink against a specific version of Python can be useful
+    if you're using py > 3.9.10 (which is currently in use in our tests github action)
+    and need to run a secondary set of tests.
```

Comment:
While this isn't technically incorrect (we are still only running Splink against versions of python <= 3.9.10 in our matrix tests), I think we should probably pre-emptively reword this.

Something along the lines of:
```suggestion
    Testing Splink against a specific version of Python, especially newer versions not included in our GitHub Actions, is vital for identifying compatibility issues 
    early and reviewing errors reported by users.
```